app.py

```diff
@@ -975,7 +994,7 @@ def chat_claude(
     socketio.emit("delete_processing_message", msg_id, room=room.name)
 
 
-def get_openai_client_and_model(model_name="NousResearch/Hermes-3-Llama-3.1-8B"):
+def get_openai_client_and_model(model_name="thewindmom/hermes-3-llama-3.1-8b"):
```

Comment:
I think we will want this to be a flag passed when we start open completion. something like `python app.py --local-activities --activity-model "thewindmom/hermes-3-llama-3.1-8b"`